mysql-test/suite/innodb/t/percona_sys_tablespace_encrypt.test

```diff
@@ -226,7 +226,6 @@ ALTER TABLE t9 TABLESPACE=`innodb_system`;
 --source include/start_mysqld.inc
 
 --remove_file $BOOTSTRAP_SQL
---let CLEANUP_FOLDER=$MYSQL_TMP_DIR/datadir1
---source include/cleanup_folder.inc
---remove_file $MYSQL_TMP_DIR/mysecret_keyring
+--force-rmdir $MYSQL_TMP_DIR/datadir1
+#--remove_file $MYSQL_TMP_DIR/mysecret_keyring
```

Comment:
Remove commented out code - but is it indeed correct, does the testcase pass the testcase check?

Reply:
I think it is bug in MTRs orphan file check.  Will remove the commented out line